.github/workflows/ci.yml

```diff
@@ -51,7 +51,7 @@ jobs:
         poetry install
     - name: Typecheck python code
       working-directory: ./python
-      run: poetry run mypy argo_client
+      run: poetry run mypy --install-types --non-interactive argo_client
```

Comment:
This change is a bit unfortunate, but perhaps unavoidable. `mypy-0.900` do not install third-party libraries' type packages unless you explicitly opt in to it with [`--install-types`](https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-install-types). Moreover, I'm unclear if it's possible to make `mypy` always use `--install-types` as a `pyproject.toml` configuration option. At the very least, I can't find anything about this in the [list of configuration options](https://mypy.readthedocs.io/en/stable/config_file.html).